crates/sui-adapter/src/execution_engine.rs

```diff
@@ -888,6 +889,35 @@ fn get_coin_balance(store: &InnerTemporaryStore, id: &ObjectID) -> u64 {
         .unwrap()
 }
 
+#[cfg(test)]
```

Comment:
nit, maybe comment about why these are now test only (just something about the mutability of shared objects needs to be specified and cannot be inferred from just the object ownership type)

Reply:
Added comment to `input_objects_from_objects`